app/assets/javascripts/projects/index.js

```diff
@@ -563,28 +561,32 @@
     });
   }
 
-  function initProjectsViewFilter() {
-    $('.projects-view-filter').click(function(event) {
-      event.preventDefault();
-      event.stopPropagation();
-      if ($(this).data('filter') === projectsViewFilter) {
-        return;
-      }
-      $('.projects-view-filter').removeClass('active');
-      $(this).addClass('active');
-      selectedProjects = [];
-      projectsViewFilter = $(this).data('filter');
-      loadCardsView();
-    });
-  }
-
   function initProjectsViewModeSwitch() {
-    $('input[type=radio][name=projects-view-mode-selector]').change(function() {
-      if (this.value === 'cards') {
+    let projectsPageSelector = '.projects-index ';
```

Comment:
remove space at end

Reply:
Woow 😱 Probably did this typo just before comitting or IDK how it was working...